radar.csv

```diff
@@ -16,4 +16,9 @@ OneDrive,adopt,tools,true,"OneDrive is our default choice for storing Office doc
 Raygun,explore,tools,true,Raygun allows loosely-coupled error reporting to a web endpoint.
 Google Drive,retire,tools,true,"We don’t have corporate Google accounts and it doesn’t make sense to subscribe to both GSuite and Office 365, both for cost reasons and from an information management point of view. The fact that we don’t have corporate accounts means data can’t be removed from Google accounts when someone leaves the company (as we don’t own the account), so we shouldn’t be storing any personal information (such as customer data) in Google Drive."
 SmartAssembly,retire,tools,false,SmartAssembly obfuscation and error reporting becomes tightly-coupled to products and slows down builds.
-Windows,adopt,platforms,true,"Our default supported operating system is Windows. If Microsoft continues to support this as part of their default support package we continue to support running our applications on that OS. See https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet"
\ No newline at end of file
+Windows,adopt,platforms,true,"Our default supported operating system is Windows. If Microsoft continues to support this as part of their default support package we continue to support running our applications on that OS. See https://support.microsoft.com/en-us/help/13853/windows-lifecycle-fact-sheet"
+Chrome,adopt,platforms,true,"Chrome is Google's evergreen browser. We support it by default"
+Firefox,adopt,platforms,true,"Firefox is Mozilla's evergreen browser. We support it by default"
+Edge,adopt,platforms,true,"Edge is Microsoft's evergreen browser. We support it by default"
+IE 11,adopt,platforms,true,"IE 11 is the default browser on Windows 10 and makes up approximately 10% of the visits to Redgate.com. We support it by default"
```

Comment:
I suggest that we put this under endure. I don't believe any new product built up should support this, unless there is a legitimate reasoning behind it. See the reasoning here: https://github.com/red-gate/Mosaic/blob/master/.adr/2018-09-13-Drop-support-for-IE-11.md

Reply:
I don't agree in this case. From a user perspective,  you've purchased Tool X using IE11, and Tool X works fine for you. Now you buy Tool Y and you've got to upgrade your browser. I think that violates people's expectations.

Don't get me wrong, I don't really want us to have to support older browsers, but we currently do and somewhere between 5%/10% of people use that browser.

Reply:
👍 happy with this if we have reasoning behind it, as we seem to. As mentioned [here](https://github.com/red-gate/Tech-Radar/issues/20#issuecomment-435363330), I am also happy for us to take the cost to support this if we believe this is the right thing for the business.

> Don't get me wrong, I don't really want us to have to support older browsers, but we currently do and somewhere between 5%/10% of people use that browser.

I guess the question here would be whether those users abandon our products if they have to switch to another browser. It may well be the case that they use it out of habit. The risk here is that we make the wrong call and potentially lose adoption/usage rate.

Other alternative would be to take the cost (which I have an assumption that get worse and worse due to IE 11 being superseded by Edge) and support IE 11 without challenging this assumption. The risk here is that this will slow us down w/o getting any business gain out of it. We are seeing examples of this type of decisions in order to move fast, see https://github.com/aspnet/Announcements/issues/324

At the end of the day, I am happy with whichever way we end up going here as I explained all the risks I can foresee for each alternative and I trust that we would make the trede-off based on these :)